Patterns/Chinese/Chinese-DateTime.yaml

```diff
@@ -85,24 +85,23 @@ DateRegexList3: !nestedRegex
   references: [MonthRegex, DayRegexNumInChinese, SimpleYearRegex, LunarRegex, WeekDayRegex, DateYearInChineseRegex, DayRegex]
 # 7/23
 DateRegexList4: !nestedRegex
-  def: '{MonthNumRegex}\s*/\s*{DayRegex}((\s+|\s*,\s*){SimpleYearRegex})?'
-  references: [MonthNumRegex, DayRegex, SimpleYearRegex]
+  def: '{MonthNumRegex}\s*/\s*{DayRegex}'
+  references: [MonthNumRegex, DayRegex]
 # 23/7
 DateRegexList5: !nestedRegex
-  def: '{DayRegex}\s*/\s*{MonthNumRegex}((\s+|\s*,\s*){SimpleYearRegex})?'
-  references: [DayRegex, MonthNumRegex, SimpleYearRegex]
+  def: '{DayRegex}\s*/\s*{MonthNumRegex}'
+  references: [DayRegex, MonthNumRegex]
 # 3-23-2017
-# The final lookahead in DateExtractor6|7|8 avoids extracting as date "10/1-11" from an input like "10/1-11/2/2017" 
```

Comment:
"10/1-11/2/2017" is not a format used. This code used to implement it has caused cases like "10/1/2018-10/7/2018" to be incorrectly extracted.

Reply:
Does it now extract cases like "10/1/2018-10/7/2018"  and "2018/1/10-2018/7/10" correctly?

Reply:
Yes. I will add them into specs.